entrypoint.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/sh
 
+# set -xv
+set +H
+
 cd "$GITHUB_WORKSPACE" || exit
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
+echo $INPUT_EXCLUDE;
+
 EXCLUDES=""
 for exclude_path in $INPUT_EXCLUDE; do
-  EXCLUDES="$EXCLUDES --exclude='!$exclude_path'"
+  EXCLUDES+=" --exclude='!$exclude_path'"
```

Comment:
This operator is not compliant with POSIX.
Either we change the interpreter to `bash` or change the way you set a variable to:
```
EXCLUDES="$EXCLUDES <set of excludes>
```